mvpa2/measures/searchlight.py

```diff
@@ -310,6 +315,11 @@ def __init__(self, datameasure, queryengine, add_center_fa=False,
         nblocks : None or int
           Into how many blocks to split the computation (could be larger than
           nproc).  If None -- nproc is used.
+        preallocate_output : bool, optional
+          If set, the output of each computation block will be pre-allocated.
+          This can speed up computations if the datameasure returns a large
+          number of samples and there are many features for which the
+          datameasure is computed.
```

Comment:
may be worth adding that since no datasets `hstack`ing would be used for them, so it will be up for a user to verify that all sa/fa returned are consistent across calls (or what happens to them in general here?)